systemtest/src/main/java/hivemall/systemtest/model/CreateTableHQ.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Hivemall: Hive scalable Machine Learning Library
+ *
+ * Copyright (C) 2016 Makoto YUI
+ * Copyright (C) 2013-2015 National Institute of Advanced Industrial Science and Technology (AIST)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package hivemall.systemtest.model;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class CreateTableHQ extends TableHQ {
+    public LinkedHashMap<String, String> header;
```

Comment:
@Nonnull final ? Why not use interface `Map<String, String>`. Implementation can change then for Map.